packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -1229,6 +1229,12 @@ mixin WidgetsBinding on BindingBase, ServicesBinding, SchedulerBinding, GestureB
   /// The `rootWidget` widget provided to this method must not already be
   /// wrapped in a [View].
   Widget wrapWithDefaultView(Widget rootWidget) {
+    assert(
+      platformDispatcher.implicitView != null,
+      'The implicitView is null. Cannot create a default View from `runApp`. '
+      'Try with `runWidget` instead.'
+      '${kIsWeb ? " See: https://docs.flutter.dev/platform-integration/web/embedding-flutter-web#embedded-mode":""}',
```

Comment:
Should we make a to redirection link for this? This will allow us to ensure that we keep the link working even if we change the website’s structure

Reply:
Yes, better, created a permalink for /to/web-multiview-runwidget here: https://github.com/flutter/website/pull/11196